integration-cli/docker_cli_swarm_test.go

```diff
@@ -855,6 +855,45 @@ func (s *DockerSwarmSuite) TestSwarmServiceTTYUpdate(c *check.C) {
 	c.Assert(strings.TrimSpace(out), checker.Equals, "true")
 }
 
+func (s *DockerSwarmSuite) TestSwarmServiceNetworkUpdate(c *check.C) {
+	d := s.AddDaemon(c, true, true)
+
+	result := icmd.RunCmd(d.Command("network", "create", "-d", "overlay", "foo"))
+	c.Assert(result.Error, check.IsNil)
```

Comment:
I might have missed this in my earlier comment. 

The value of using `icmd.RunCmd()` is that you get a full error message with all the details (expected vs actual). The value is lost if all that's being checked is the error.

These `c.Assert(result.Error, check.IsNil)` should be replaced with `result.Assert(c, icmd.Success)`.